e2e_tests/data/yolov8/to/test_input_with_bboxes_and_polys.txt

```diff
@@ -1,3 +1,4 @@
 0 0.59466 0.112 0.9053199999999999 0.112 0.9053199999999999 0.32265999999999995 0.59466 0.32265999999999995
```

Comment:
This change is necessary after fixing the polygon > YOLOv8 conversion issue above - This is the corrected version of the output file